crt_portal/cts_forms/widgets.py

```diff
@@ -5,3 +5,31 @@ class UsaRadioSelect(ChoiceWidget):
     input_type = 'radio'
     template_name = 'django/forms/widgets/radio.html'
     option_template_name = '../templates/forms/widgets/usa_radio_option.html'
+
+
+# Overrides Django CheckboxSelectMultiple:
+# https://docs.djangoproject.com/en/2.2/ref/forms/widgets/#checkboxselectmultiple
+class UsaCheckboxSelectMultiple(ChoiceWidget):
```

Comment:
Could be an improvement if this inherits from `CheckboxSelectMultiple` instead of `ChoiceWidget`...